### ISSUE TITLE
[#SomeChanges Suggestion] PvP gearing time frame - honor

### ISSUE THREAD

**Issue by leekzy** [ClassicWoWCommunity/cata-classic-bugs#1293]
#### Conquest Problem
*Already addressed* ❤

#### Honor Problem
The current system for obtaining a full honor set poses several challenges, particularly given the evolving expectations and habits of modern-day players.

**Use Case:**
Modern players are significantly more alt-prone, often maintaining multiple characters to explore different classes and roles. This is a shift from earlier playstyles, where players typically focused on one main character. The existing system, with its slow honor gain, fails to meet the expectations set by previously available farming methods in Classic, where players had more efficient ways to gear up multiple characters.

**Barriers to Entry:**
The high cost and slow accumulation of honor points create a substantial barrier to entry for new players and alts, which is counterproductive for the health of PvP communities. This prolonged gearing process can deter players from participating in arenas, a space that thrives on diverse and frequent participation.

**Details:**
- **Full honor set cost:** Approximately 27k Honor, including shoulder and helm resilience enchants.
- **Daily BG win rewards:** Initially 380-400 honor, later dropping to 230-250.
- **Honor per win:** 250
- **Honor per loss:** 125
- **Average honor per BG with 50% win rate:** (250 + 125) / 2 = 187.5
- **Additional honor from kills per BG/daily bonus parsed out:** Estimated average of 50 honor per BG
- **Total average honor per BG:** 187.5 + 50 = 237.5
- **Total honor required:** 27,000

- **Number of BGs required with 50% win rate:** 
  - 27,000 / 237.5 ≈ 114 BGs

- **Time investment for 114 BGs:** 
  - 114 * 15 minutes ≈ 28.5 hours

- **Average queue time and breaks between BGs:** Assume 3 minutes per BG
- **Total time including queue times:** 
  - 114 * (15 + 3) minutes ≈ 34 hours

**Tol Barad Honor Gain:**
Previously, Tol Barad was an extremely time-efficient way to farm honor, offering up to 1800 honor points per win for attackers. This significantly reduced the grind and provided a faster alternative to battlegrounds ([source](https://www.mmo-champion.com/threads/819922-Tol-Barad-mow-much-honor-for-winning), [source](https://www.mmo-champion.com/threads/821732-Ideas-to-REALLY-fix-Tol-Barad), [source](https://www.mmo-champion.com/threads/824117-Tol-Barad-Honor-Change-Bridge-Exploit-Jan-3-Hotfixes)).

**Proposed Changes and New Grind Estimate:**
- **Buffed Honor Rate:** Increasing honor rate gain by 2x-3x.
  - If honor rate is increased by 2x, average honor per BG would be 475.
  - If honor rate is increased by 2.5x, average honor per BG would be 593.75.
  - If honor rate is increased by 3x, average honor per BG would be 712.5.
  
- **Revised Estimated BGs Required:** 
  - With a 2x honor rate increase, the number of BGs required would be approximately 57.
  - With a 2.5x honor rate increase, the number of BGs required would be approximately 46.
  - With a 3x honor rate increase, the number of BGs required would be approximately 38.
  
- **Revised Time Investment:**
  - At a 2x honor rate, the time investment would drop to around 14.5 hours (without queue times).
  - At a 2.5x honor rate, the time investment would drop to around 11.5 hours (without queue times).
  - At a 3x honor rate, the time investment would drop to around 9.5 hours (without queue times).

- **Including queue times:**
  - At a 2x honor rate, the time investment would drop to around 17 hours.
  - At a 2.5x honor rate, the time investment would drop to around 14 hours.
  - At a 3x honor rate, the time investment would drop to around 11.5 hours.

^obviously there is some napkin math and it's more dynamic, but it should give an estimation about current time investment to get going in PvP per character.

**Additional Considerations:**
As MannySJH pointed out: "There might be some unforeseen consequences with increasing honor gains since you can trade honor for crafting materials and justice points later on. Probably better to just reduce the cost of items similar to what happened in wrath classic." I agree with this, and looking into reducing cost of honor items with %based reductions based on the above numbers is perhaps an even better way to go about it.

- **Honor Weapons:** Introducing honor weapons could provide an additional incentive for PvP participation. However, to prevent any negative impact on PvE balance, these weapons could be zone-locked to arenas or battlegrounds. This would ensure they remain a reward strictly for PvP efforts without disrupting PvE progression.

I'm hoping that by making these adjustments, we can create a more welcoming and engaging PvP environment that caters to the modern player's needs and encourages broader participation in arenas and battlegrounds.

**Comment by Kenniflopper** [comment]
Old post as reference - https://github.com/JamminL/cata-classic-bugs/issues/951

**Comment by MannySJH** [comment]
There might be some unforeseen consequences with increasing honor gains since you can trade honor for crafting materials and justice points later on. Probably better to just reduce the cost of items similar to what happened in wrath classic.

**Comment by leekzy** [comment]
> There might be some unforeseen consequences with increasing honor gains since you can trade honor for crafting materials and justice points later on. Probably better to just reduce the cost of items similar to what happened in wrath classic.

Good point, that is definitely a great suggestion as well, will update post

**Comment by JamminL** [comment]
Archiving old non-relevant issues in preparation for MOP. If I made a mistake please comment on it.
